DCPvsOID/DCPvsOID.md

```diff
@@ -4,14 +4,18 @@ _Areas of application, commonalities and differences, and application in Dataspa
 ## Executive Summary
 
 This document delves into the comparative analysis of the [Eclipse Decentralized Claims Protocol (DCP)](https://projects.eclipse.org/projects/technology.dataspace-dcp)  and the [OpenID4VC Protocol (OID4VC)](https://openid.net/sg/openid4vc/). Both protocols serve essential roles in enabling secure, interoperable, and privacy-preserving data exchanges within digital ecosystems, yet their implementation diverges in meaningful ways. Understanding these contrasts is key to making informed decisions about which protocol to adopt for specific requirements within dataspaces.
-It shows that the Eclipse DCP is the best solution for providing organizational credentials through dataspace connectors.
+In a dataspace, it'll be the responsability of the Dataspace Authority to decide which of the protocol suits better their interoperablity, policy and regulatory compliance requirements.
 
 ## Introduction
-Trust in dataspaces is created through the reconciliation of policies and claims containing evidence which support the required policies — a process that ensures all participating entities align with shared governance principles, operational standards and that access, contract and usage policies of data sharing contracts are met.
+Trust in dataspaces is created through the reconciliation of policies, claims and evidence which support the required policies — a process that ensures all participating entities align with shared governance principles, operational standards and that access, contract and usage policies of data sharing contracts are met.
 
 Policies, which define the rules and expectations within the dataspace, serve as the foundation for establishing trust. Claims, on the other hand, are assertions made by entities about their compliance with these policies or their possession of certain attributes.
+At last, Evidences bring additional information to support one or more claims and facilitate the risk-assessment made by the participating entities.
 
```

Comment:
Evidence is depending on the trust framework and corresponding trust anchors. IMHO it is not the responsibility of the protocol to assess the credentials origin. I doubt this aspect should be addressed within the document at hand. However, we may discuss about this as it is ofc an important aspect from the overall concept and there are relations to the profile specification projects we have in EDWG.